modules/mapreduce/pom.xml

```diff
@@ -59,17 +51,34 @@
       <scope>test</scope>
     </dependency>
     <dependency>
-      <groupId>org.apache.accumulo</groupId>
-      <artifactId>accumulo-minicluster</artifactId>
-      <scope>test</scope>
-    </dependency>
-    <dependency>
       <groupId>org.apache.fluo</groupId>
       <artifactId>fluo-integration</artifactId>
       <type>test-jar</type>
       <scope>test</scope>
     </dependency>
   </dependencies>
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-dependency-plugin</artifactId>
+        <version>2.10</version>
+        <executions>
+          <execution>
+            <id>analyze</id>
+            <goals>
+              <goal>analyze-only</goal>
+            </goals>
+            <configuration>
+              <ignoredDependencies combine.children="append">
+                <ignoredDependency>org.apache.accumulo:accumulo-minicluster:jar:1.7.1</ignoredDependency>
```

Comment:
TravisCI merges a PR to master and then runs the tests.   In some cases this may cause the build to fail when it does not fail in the branch.  For example the difference in Accumulo versions between master and this branch.